### DISCUSSION TITLE
How to hide a menu?

### DISCUSSION THREAD

**Discussion by pH142857** [thuliteio/doks#987] [General]
I have a first-level menu element:
```
[[docs]]
  name = "hello there"
  weight = 200
  identifier = "hello-there"
  url = "/docs/hello-there/"
```
and in folder /docs/hello-there, I have many MD files with front matter:
```
menu: 
  docs:
    parent: "hello-there"
```

These elements are not ready yet, so I don't want them to appear in my doc. However I don't wanna delete them either. **How can I hide them?** I tried to #comment out the menu element but the front matter makes the menu appear anyway, with the "hello-there" first-level label instead of the "Hello there" one.

[This page](https://getdoks.org/docs/how-to/pages-and-menus/menus/) didn't help.

**Comment by h-enk** [comment]
Would setting `draft: true` in the frontmatter of these pages work for you?

Or, set Hugo's  [Build Options](https://gohugo.io/content-management/build-options/)?

**Reply by pH142857** [reply]
Sure does! I hadn't seen this option. An even better fit would be a way to deactivate the whole first-level menu, because going through each page is tedious, but that will do anyway. Thanks!

**Reply by h-enk** [reply]
Setting `draft: true` in the first level `_index.md` only should be sufficient

**Reply by pH142857** [reply]
Holy cows, thanks a lot!
